Remotion/Web/Core/Res/Themes/NovaViso/HTML/Common.css

```diff
@@ -514,8 +518,22 @@ img.disabled,
 .remotion-themed.scrollable
 {
   overflow: auto;
-  scrollbar-width: var(--remotion-themed-scrollbar-style);
-  scrollbar-color: var(--remotion-themed-scrollbar-color);
+}
+
+.remotion-themed.scrollable > *
+{
+  scrollbar-color: initial; /* Stop inheriting the scrollbar color to child elements */
+  scrollbar-width: initial; /* Stop inheriting the scrollbar width to child elements */
+}
+
```

Comment:
Why did you move this one up in the file? it's a deeper nested selector so semantically "after" the others.